src/lib/structures/expr.ml

```diff
@@ -2115,6 +2115,16 @@ module Triggers = struct
         if c <> 0 then c else Var.Set.compare y1 y2
     end)
 
+  (* let's [e] be a subterm of a quantified formula [f] and [bv] the set
```

Comment:
```suggestion
  (* let [e] be a subterm of a quantified formula [f] and [bv] the set
```

Reply:
I think that mentioning the formula `f` is confusing here since it is not an argument of the function (and in full generality might not exist). I'd say something along the lines of:

> [underscore bv e] substitutes all free (term) variables in [e] that are not in [bv] by the special `underscore` variable.

and give the example of `g(x, y, z)` with `bv = {x, y}` without mentioning a formula at all.

Reply:
Sure, I wasn't happy with this formulation too.